modules/apps/translation/translation-google-cloud-translator/src/main/java/com/liferay/translation/google/cloud/translator/internal/translator/GoogleCloudTranslator.java

```diff
@@ -151,8 +160,14 @@ protected void activate(Map<String, Object> properties) {
 			ConfigurableUtil.createConfigurable(
 				GoogleCloudTranslatorConfiguration.class, properties);
 
-		if (!isEnabled()) {
-			return;
+		try {
+			if (!isEnabled(CompanyThreadLocal.getCompanyId())) {
```

Comment:
@adolfopa 
On this case I do not know how can we obtain the companyId 
Have you any advice? or we use the deprecated isEnabled()?

Thanks!

Reply:
We need to move the code inside active to a new private method, and get a new (google) Translate object on each call. In the future we may be able to implement some caching mechanism to avoid too many reinitializations, but for the moment I would leave it like that.